1010/en.md

```diff
@@ -0,0 +1,30 @@
+Chess board knights only travel in L shaped squares. 
```

Comment:
Add problem title like this: https://github.com/lightoj-dev/problem-tutorials/pull/11/files#diff-49d89fcefb3733849902cae167206bfef3d2d2fa1569a2abcc5b303e53f828b9R1